name.abuchen.portfolio/src/name/abuchen/portfolio/model/ClientFactory.java

```diff
@@ -1170,6 +1174,29 @@ private static void fixDimensionsList(Client client)
         });
     }
 
+    private static void fixSourceAttributeOfTransactions(Client client)
+    {
+        List<Transaction> allTransactions = new ArrayList<>();
+        client.getAccounts().forEach(a -> allTransactions.addAll(a.getTransactions()));
+        client.getPortfolios().forEach(p -> allTransactions.addAll(p.getTransactions()));
+
+        Pattern pattern = Pattern.compile("^((?<note>.*) \\| )?(?<file>[^ ]*\\.(pdf|csv))$"); //$NON-NLS-1$
+
+        for (Transaction tx : allTransactions)
+        {
+            String note = tx.getNote();
+            if (note == null)
+                continue;
+
+            Matcher m = pattern.matcher(note);
+            if (m.matches())
+            {
+                tx.setNote(m.group("note")); //$NON-NLS-1$
```

Comment:
`tx.setNote(m.group("note")); //$NON-NLS-1$`
to
tx.setNote(m.group("note").trim()); //$NON-NLS-1$